test/rowSelection.test.tsx

```diff
@@ -184,6 +185,109 @@ test('extra keys are preserved when updating the selectedRows Set', () => {
   expect(set).toStrictEqual(initialSet);
 });
 
+test('toggling selection of row groups', () => {
```

Comment:
Do we have duplicate tests?
https://github.com/adazzle/react-data-grid/blob/main/test/grouping.test.tsx#L238